WooCommerce/UITestsFoundation/Screens/Products/SingleProductScreen.swift

```diff
@@ -30,4 +30,20 @@ public final class SingleProductScreen: ScreenObject {
 
         return self
     }
+
+    public func addProductTitle(productTitle: String) throws -> Self {
+        app.cells["product-title"].tap()
+        app.cells["product-title"].typeText(productTitle)
+        return self
+    }
+
+    public func publishProduct() throws -> Self {
+        app.buttons["publish-product-button"].tap()
+        return self
+    }
+
+    public func verifyNewProductScreenLoaded() {
+        XCTAssertTrue(app.buttons["save-product-button"].exists)
+        XCTAssertTrue(app.staticTexts["TIP"].exists)
```

Comment:
WDYT of checking that this screen contains the product name?

Reply:
updated in https://github.com/woocommerce/woocommerce-ios/pull/8380/commits/91b15ef1f55c562f15434b7d1f7e8f3459308652#diff-38e7198d7b53c64cf1257404887931cc42ac178877c05d5bd98a463ac1da1d0b